daniellamarr/fastfoodfast-live
# #160986354 User should get all available menu

#### What does this PR do?
- This PR gets all available menu on the app
#### Description of Task to be completed?
- When a user navigates to **index.html**, the user should be able to get all available menu on the app
- A successful request should return a status of 200
#### How should this be manually tested?
- Pull this branch **ft-user-able-get-all-available-menu-160986354** off this repository
- Navigate to **index.html** on the branch
- A user should see all available menu on the app
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
[160986354](https://www.pivotaltracker.com/story/show/160986354)
#### Screenshots (if appropriate)

#### Questions:
None
